SoftLayer/utils.py

```diff
@@ -459,3 +461,74 @@ def decode_stacked(document, pos=0, decoder=JSONDecoder()):
         obj, pos = decoder.raw_decode(document, pos)
 
         yield obj
+
+
+def console_color_themes(theme):
+    """Colors in https://rich.readthedocs.io/en/stable/appendix/colors.html?highlight=light_pink1#standard-colors"""
+    # Default theme
+    if not theme:
+        return Console(theme=Theme(
+            {
+                "options": "bold cyan",  # OPTIONS
+                "command": "orange3",  # COMMAND
+                "args": "bold cyan",  # ARGS
+                "path": "bold red",  # command path
+                "name_sub_command": "orange3",  # sub command name
+                "sub_command": "orange3",  # sub command list
+                # Help table colors options
+                "option": "bold cyan",
+                "switch": "bold green",
+                "default_option": "light_coral",
+                "option_keyword": "bold cyan",
+                "args_keyword": "bold green",
+            })
+        )
```

Comment:
We dont need to re-define the dark theme here.

basically just refactor this to be 

```python

if theme == 'light':
    # return the light theme console
else:
    # return the dark theme console
```

Reply:
Done